src/Consolonia.Core/Drawing/RenderTarget.cs

```diff
@@ -24,10 +24,12 @@ internal class RenderTarget : IDrawingContextLayerImpl
         // cache of pixels written so we can ignore them if unchanged.
         private Pixel?[,] _cache;
         private ConsoleCursor _consoleCursor;
+        private FlushingBuffer _flushingBuffer;
```

Comment:
Do we move FlushingBuffer to field to save on object allocation? This does not give anything: FlushingBuffer is a struct itself, while BufferBuffer flushing just drops the reference to string data, which goes to GC same way.